src/storage/storage.intern.h

```diff
@@ -118,6 +118,10 @@ typedef struct StorageInterfaceNewReadParam
 
     // Limit bytes read from the file. NULL for no limit.
     const Variant *limit;
+
+    // File version to read (NULL for current version)
```

Comment:
nit: maybe have this right before the `String` def?

Reply:
The comment dates from before the boolean was added, so redid the comments in 3f02a3fea9e.